lib/graphql_schema.rb

```diff
@@ -62,7 +62,14 @@ def deprecation_reason
     end
   end
 
+  module Description
```

Comment:
It looks like everything with a name also has a description, so it might just make sense to put the description in the NamedHash module